components/IscnRegisterForm.vue

```diff
@@ -308,16 +334,35 @@
                 {{ formattedUploadStatus }}
               </div>
             </div>
-            <Button
-              v-else
-              :text="$t('IscnRegisterForm.button.register')"
-              type="submit"
-              preset="secondary"
+            <div v-else
+              :class="[
+                'flex',
+                'flex-col',
+                'items-end',
+              ]"
             >
-              <template #append>
-                <IconArrowRight />
-              </template>
-            </Button>
+              <Button
+                :text="$t('IscnRegisterForm.button.register')"
+                type="submit"
+                preset="secondary"
+              >
+                <template #append>
+                  <IconArrowRight />
+                </template>
+              </Button>
+              <a
+                :class="[
+                  'text-[8px]',
+                  'underline',
+                  'mt-[4px]',
+                  'mb-[-12px]',
+                ]"
+                href="#"
+                @click.prevent="showUploadOnly = true"
+              >
+                {{ $t('IscnRegisterForm.button.uploadOnly') }}
+              </a>
```

Comment:
I believe this button should be moved to the left, as the main action should always remain on the right-most side

Also, this `<a/>` could be replaced by `<Button preset="outline" />` to ensure consistent styling

Reply:
> I believe this button should be moved to the left, as the main action should always remain on the right-most side
> 
> Also, this `<a/>` could be replaced by `<Button preset="outline" />` to ensure consistent styling

Totally missed there are 2 parts of review..
<img width="620" alt="image" src="https://github.com/likecoin/app-like-co/assets/6198816/b419622b-6fef-44e2-8d06-746d7b39a066">

Does this look ok?